### PR TITLE
ft: expose rclone tunables

### PR DESCRIPTION
**Pull request by ssalaues**
**What does this PR do, and why do we need it?**
Exposes rclone flags and tunables to the values yaml file. While it isn't necessary to modify these values, it does provide a great deal of tune-ability to the chart.

**Which issue does this PR fix?**

fixes ZENKO-1561

**Special notes for your reviewers**:
Does not change behavior and more than half the changes are doc updates

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

File: docs/docsource/Installation/topics/adding_NFS_storage.rst [hunk 1]
```diff
@@ -38,76 +38,78 @@ and their default values.
...
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.service.type``                 | pfsd service type                     | ``ClusterIP``                |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.service.port``                 | pfsd service port                     | ``80``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.resources``                    | pfsd resource requests and limits     | ``{}``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.nodeSelector``                 | Node labels for pfsd pod assignment   | ``{}``                       |
+   +---------------------------------------+---------------------------------------+------------------------------+
+   | ``pfsd.tolerations``                  | Node taints to tolerate               | ``[[``                       |
```
Comment (alexanderchan-scality) [inline docs/docsource/Installation/topics/adding_NFS_storage.rst]: minor?
```suggestion
   | ``pfsd.tolerations``                  | Node taints to tolerate               | ``[]``                       |
```

**Review by alexanderchan-scality — APPROVED** [review]

File: docs/docsource/Installation/topics/adding_NFS_storage.rst (hunk 1 above)
Comment (ssalaues) [reply docs/docsource/Installation/topics/adding_NFS_storage.rst]: Yeah but I'll fix it. Thanks for the find!

**Comment by ssalaues** [comment]
@bert-e approve

**Comment by ssalaues** [comment]
@bert-e reset

**Comment by ssalaues** [comment]
@bert-e status

**Pull request merged**